src/Forms/ConfirmedPasswordField.php

```diff
@@ -552,8 +590,6 @@ public function validate($validator)
     }
 
     /**
-     * Only save if field was shown on the client, and is not empty.
-     *
```

Comment:
Isn't the "Only save if field was shown on the client" part still accurate? And "and is not empty" is still _sometimes_ accurate.
We should probably add _more_ context here, not remove what context was already there.

Reply:
It's just an implementation of the standard FormField::saveInto() method, I don't think the docblock should have "context". You should just like you any other saveInto() method

Reply:
I disagree in this case, because the record is only saved into _under certain circumstances_ which differs from how this method is used in other fields.

This API is surfaced in IDEs as well as in the api docs in api.silverstripe.org so it should be retained.

Reply:
Updated